backend/backend/settings.py

```diff
@@ -114,7 +115,16 @@
 # Internationalization
 # https://docs.djangoproject.com/en/1.11/topics/i18n/
 
-LANGUAGE_CODE = 'en-us'
+LANGUAGES = (
+    ('en', 'English'),
+    ('de', 'Deutsch'),
```

Comment:
We had a discussion about language labels. We discussed if we want to use the 2 or 5 character format ('en' vs 'en-us'). As far as I remember, we said that 2 characters are not sufficient. I tried to find the conversation in Slack without success.

Reply:
This only handles the translation of the backend and does not involve the content. I thought we agreed on supporting only English and German on the mid-term and maybe adding a few more languages in the future.
But it's nearly no effort to change this to 5 chars just to be completely future-proof.

Reply:
Alright, then my comment might not be that relevant. One other question: does it make sense to integrate the content translations and back end translations somehow? This means, for example, that content languages are "the same" as back end interface languages (and not decoupled).

Reply:
I wouldn't recommend merging them; there's just no benefit as far as i can see and might add confusion.

Also we agreed on the bcp47 specification for content languages, which allows quite a lot of formats (to be future proof, if we want to support region-codes and / or "einfache Sprache" ): https://tools.ietf.org/html/bcp47

Reply:
Yes, I agree that we should not mix the two levels up - we don't want to handle the content's translations in .po files and sometimes we want to edit an english page and keep the backend in german...